string component_id
---
string status
bool success
string message 